### PR TITLE
README: Fix new method signatures using the request parameter.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Comment by Natim** [comment]
@googlebot I signed it!

File: README.rst [hunk 1]
```diff
@@ -138,8 +138,7 @@ the topic, and subscribe to that, passing a callback function.
         project_id=os.getenv('GOOGLE_CLOUD_PROJECT'),
         sub='MY_SUBSCRIPTION_NAME',  # Set this to something appropriate.
     )
-    subscriber.create_subscription(
-        name=subscription_name, topic=topic_name)
+    subscriber.create_subscription(request={"name": subscription_name, "topic": topic_name})
```
Comment (plamut) [inline README.rst:141]: Same thing here - `create_subscription()` supports subscription name and topic name as keyword arguments ([source](https://github.com/googleapis/python-pubsub/blob/7228f6c9a4c050bf22bb4bc3582b89b04eaa8702/google/pubsub_v1/services/subscriber/client.py#L298-L309)).

File: README.rst [hunk 2]
```diff
@@ -110,7 +110,7 @@ messages to it
         project_id=os.getenv('GOOGLE_CLOUD_PROJECT'),
         topic='MY_TOPIC_NAME',  # Set this to something appropriate.
     )
-    publisher.create_topic(topic_name)
+    publisher.create_topic(request={"name": topic_name})
```
Comment (plamut) [inline README.rst:113]: The signature here is fine - this particular method supports subscription name and topic name as keyword arguments ([source](https://github.com/googleapis/python-pubsub/blob/7228f6c9a4c050bf22bb4bc3582b89b04eaa8702/google/pubsub_v1/services/publisher/client.py#L277-L285)).

**Review by plamut — COMMENTED** [review]
Thank you for your kind pull request!

It does appear, though, that the example in README does not have to be fixed - several methods can be called with kwargs only, it is not necessary to use the `request` argument.

More [details](https://github.com/googleapis/python-pubsub/blob/7228f6c9a4c050bf22bb4bc3582b89b04eaa8702/UPGRADING.md#more-details) can be found in the UPGRADING guide.

**Comment by Natim** [comment]
> More details can be found in the UPGRADING guide.

Ok so I just need to name the parameters. Thank you for the precisions.

**Pull request closed without merging**